client/src/document/index.scss

```
@@ -83,9 +83,10 @@
   }
 
   img {
-    background: var(--background-primary);
+    background: #fff;
     border: 1px solid var(--border-primary) !important;
     border-radius: var(--elem-radius);
+    color: #000;
```

Let's use the same color as text in the light theme, and add a comment:

```suggestion
    /* Required for alt texts. */
    color: #1b1b1b;
```